src/adaptors/merkl/merkl-additional-reward.js

```diff
@@ -0,0 +1,68 @@
+const { networks } = require('./config');
+const { getData } = require('../utils');
+
+exports.addMerklRewardApy = async (
+  pools,
+  protocolId,
+  poolAddressGetter,
+) => {
+  try {
+    let merklPools = [];
+    let pageI = 0;
+
+    while(true) {
+      let data;
+      try {
+        data = await getData(`https://api.merkl.xyz/v4/opportunities?mainProtocolId=${protocolId}&status=LIVE&items=100&page=${pageI}`);
```

Comment:
is empty for `gamma`, can u confirm this is correct?

Reply:
You are correct
`gamma` is the correct id (checked by using status=PAST`)

There is no current incentive for Gamma but they might restart, I'm happy to drop it for now and add it back when incentive starts again?

Reply:
I've removed Gamma
Incentive are done differently now, I will open a separate PR